frontend/src/App.test.js

```diff
@@ -11,12 +12,28 @@ beforeAll(() => {
   global.Date.now = mockedDate.now;
 });
 
+jest.mock('axios');
+
 describe('App', () => {
   it('should be defined', () => {
     expect(App).toBeDefined();
   });
+
   it('should render correctly', () => {
     const tree = renderer.create(<App />).toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  it('should initiate GET request using axios() to servlet', () => {
+    const servletEndpoint = '/api/v1/search-match';
+    const mockMatchStatus = {matchStatus: 'true'};
+    const mockData = {data: mockMatchStatus};
+
+    axios.get.mockImplementation(() => Promise.resolve(mockData));
+    const responseData = axios.get(servletEndpoint);
```

Comment:
This test should interact with the App in some way rather than mocking the network response and then sending the request from the test. Otherwise this test is only testing its own setup.

Reply:
Oh good point, changed it to interact with App instead of testing setup.

Reply:
forgot to push?